### PR TITLE
add yolo_v3 to 2.0

### PR DESCRIPTION
**Pull request by changwangss**
Signed-off-by: Wang, Chang1 <chang1.wang@intel.com>

## Type of Change

add yolo_v3 success
performance issue: 
wav2vec2_fx: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation/239426
maskrcnn_fx: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation/239424/
trace failed: rnnt

## Description

detail description 
JIRA ticket: xxx
https://jira.devtools.intel.com/browse/ILITV-2625
## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4204/

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4206/

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4207/

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4211/
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4212/

File: examples/pytorch/object_detection/maskrcnn/quantization/ptq_static/fx/MASKRCNN_README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# 1. Problem
```
Comment (PenghuiCheng) [inline examples/pytorch/object_detection/maskrcnn/quantization/ptq_static/fx/MASKRCNN_README.md:1]: Please follow README format as other examples

File: examples/pytorch/object_detection/maskrcnn/quantization/ptq_static/fx/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+# Prerequisite
+
+### 1. Environment
+
+PyTorch >=1.8 and <=1.11 version is needed.
+
+```shell
+cd examples/pytorch/object_detection/maskrcnn/quantization/ptq/fx
+pip install -r requirements.txt
```
Comment (PenghuiCheng) [inline examples/pytorch/object_detection/maskrcnn/quantization/ptq_static/fx/README.md:14]: requirements.txt path is error

File: examples/pytorch/object_detection/maskrcnn/quantization/ptq_static/fx/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+neural-compressor
+yacs
+--find-links https://download.pytorch.org/whl/torch_stable.html
```
Comment (PenghuiCheng) [inline examples/pytorch/object_detection/maskrcnn/quantization/ptq_static/fx/requirements.txt:3]: This url seems to be deprecated. let's remove it.

File: examples/pytorch/object_detection/maskrcnn/quantization/ptq_static/fx/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+torch<=1.11.0+cpu
```
Comment (PenghuiCheng) [inline examples/pytorch/object_detection/maskrcnn/quantization/ptq_static/fx/requirements.txt:4]: can we use latest version of torch and torchvision?

**Comment by changwangss** [comment]
yolo_v3: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4216/

**Review by PenghuiCheng — APPROVED** [review]

**Review by PenghuiCheng — APPROVED** [review]

**Pull request merged**